index.md

```diff
@@ -130,27 +110,12 @@ You may wish to consider the following items when selecting an evaluation tool:
 
 ### Organizational Structure and Development Process
 
-Larger organizations may gain from using a combination of evaluation tools. Using a combination of tools could meet the various teams' needs during all stages of the project.
+Development teams may gain from using a combination of evaluation tools. Using a combination of tools could meet the various teams members' needs during all stages of the project.
 
-### Complexity and Size of the Web Site
+### Complexity and Size of the Web Content
 
 Some complex sites could have a lot of multimedia content. Other complex sites could use advanced technologies such as SMIL or MathML. Specialized evaluation tools may be more useful for these sites, even if they may be more limited. 
 
 ### Skills and Knowledge of the Web Developers
 
 Select the tool commensurate to the developer's skills. Some evaluation tools require users to have more knowledge of accessibility or code. Developers may also use some tools to increase their accessibility or code knowledge.
-
-## Related Pages
-{:#related}
-
-
-
-
-
-
-
-
-
-This document is part of a multi-page [Evaluating Web Accessibility
-resource suite](#) that outlines different approaches for
-evaluating Web accessibility.
```

Comment:
I think this cross-link is adequately addressed by the new WAI design, so that this section is no longer needed.